cypress/integration/dashboardPremium.js

```diff
@@ -4,7 +4,7 @@ describe('Dashboards Premium Features', () => {
         cy.location('pathname').should('include', '/dashboard')
     })
 
-    it('Tag dashboard', () => {
+    xit('Tag dashboard', () => {
```

Comment:
These tests passing in the past are a red herring, since taggables on dashboards should be an ee only feature. We don't have ee actually setup in cypress and these will fail now that we check in the backend now and surface a 402 error.

Reply:
Good point, adding a comment here